src/configurator.rb

```diff
@@ -112,7 +110,11 @@ def initialize(identifier:, properties:, configure_file:,
       def ask(highline)
         ask_method = "ask_#{type}_question"
         self.send(ask_method, highline) do |highline_question|
-          highline_question.default = default
+          if !default.to_s.empty?
```

Comment:
This is possibly a roundabout/over-complex way to test that a default has been given - do you think we can just use [`if default.present?`](http://guides.rubyonrails.org/active_support_core_extensions.html#blank-questionmark-and-present-questionmark) and get the same effect?